contracts/contract-webauthn-secp256r1/src/lib.rs

```diff
@@ -219,20 +188,20 @@ impl CustomAccountInterface for Contract {
         signature: Signature,
         auth_contexts: Vec<Context>,
     ) -> Result<(), Error> {
-        // Only the sudo signer can `add_sig`, `rm_sig`, `resudo` and `upgrade`
+        // Only the super signer can `add_sig`, `rm_sig`, `re_super` and `upgrade`
         for context in auth_contexts.iter() {
             match context {
                 Context::Contract(c) => {
                     if c.contract == env.current_contract_address()
                         && (c.fn_name == Symbol::new(&env, "add_sig")
                             || c.fn_name == Symbol::new(&env, "rm_sig")
-                            || c.fn_name == Symbol::new(&env, "resudo")
+                            || c.fn_name == Symbol::new(&env, "re_super")
                             || c.fn_name == Symbol::new(&env, "upgrade"))
                     {
                         if env
                             .storage()
                             .instance()
-                            .get::<Symbol, Bytes>(&SUDO_SIGNER)
+                            .get::<Symbol, Bytes>(&SUPER)
                             .ok_or(Error::NotFound)?
                             != signature.id
                         {
```

Comment:
`.extend_ttl(&signature.id, max_ttl, max_ttl);` - I'm not sure if always setting the threshold to max ttl is a good idea; it makes frequent transactions to unnecessarily spend a bit more fees on extension without much benefit (say, your first transaction extends the contract to 120 days, then the second transaction extends it for 10 more minutes. is it really worth it?). I'd suggest using max_ttl - a few days in order to reduce the bump frequency. That's worth changing across all the TTL extensions in these contracts.

Reply:
Made some adjustments, is that what you had in mind? Max extend but with a 7 day max threshold.

Reply:
Yeah, that's good